docs/comments_endpoints.md

```diff
@@ -72,11 +128,52 @@ Response
 ```ruby
 Status: 201
 {
-  message: "Successfully added"
+  id: 4,
+  text: 'This is such a scam',
+  date_created: 'Wed, 25TH Nov, 2017 12:00PM',
+  updated: true,
+  score: 9,
+  user: {
+    id: 3,
+    name: 'Bayo Owoade'
```

Comment:
I don't know, but would it be possible to update all the docs to include all the json data we'd be returning. e.g for each of the resources I guess we'd be returning the user object with the profile :id, :name, :email, :image, :points etc.